assets/styles/pages/homepage.less

```diff
@@ -196,7 +202,7 @@ font-size: @font-size-homepage-normal;
 
   .product-description {
```

Comment:
The font size of the new `p` section here feels a bit large and out of place.

I think if we added this it could make it look a little cleaner:
```
p {
  font-size: @font-size-homepage-product-description;
  margin: auto;
  width: 85%;
}
```

Reply:
Oh the `h2` here also probably doesn't need as much bottom margin now that there's more text underneath it. So I think adding this too could help.
```
h2 {
  margin-bottom: @margin-md;
}
```